website/docs/reference/project-configs/clean-targets.md

```diff
@@ -23,31 +23,31 @@ If this configuration is not included in your `dbt_project.yml` file, the `clean
 :::info
 This is our preferred configuration
 :::
-To remove packages as well as compiled files, include the value of your [modules-path](modules-path) configuration in your `clean-targets` configuration.
+To remove packages as well as compiled files, include the value of your [packages-install-path](packages-install-path) configuration in your `clean-targets` configuration.
```

Comment:
I think a `<Changelog>` here would make a lot of sense, since it will require active change from users when they upgrade to v1